test/test_regression.py

```diff
@@ -246,3 +264,13 @@ def test_fetch():
                 first_row_data = list(data_frame[table_i].iloc[0])
                 assert expected_data == first_row_data
                 assert expected_column_values == column_values
+
+
+def test_fetch_order():
```

Comment:
I think we can add more cleaning up.
use` @pytest.mark.parametrize("dataset, expected", fetch_order_tests)`
This should be the same as the for loop. `for dataset, expected in fetch_order_tests:`
Something like 

```python
@pytest.mark.parametrize("dataset, expected", fetch_order_tests)
def test_fetch_order():
    """Test fetch dataframe order"""
        data_frame_dict = fetch(dataset)
        assert data_frame_dict.keys() == expected
```
dict.keys returns list of keys and since we return ordered data_frame_dict should be order preserved.